dev/start

```diff
@@ -256,6 +257,24 @@ env_args="$env_args -e CONJUR_AUTHENTICATORS=$enabled_authenticators"
 
 docker-compose up -d --no-deps $services
 
+if [[ $ENABLE_AUDIT = true ]]; then
```

Comment:
This block is a duplicate of https://github.com/cyberark/conjur/blob/de82a4a4b594a9cca62a0f475eb5f3f8cacc1c15/dev/start#L120-L127

Is this intentional, or the result of a bad rebase?

Reply:
bad rebase